src/include/duckdb/parser/column_definition.hpp

```diff
@@ -28,6 +28,8 @@ class ColumnDefinition {
 	DUCKDB_API ColumnDefinition(string name, LogicalType type);
 	DUCKDB_API ColumnDefinition(string name, LogicalType type, unique_ptr<ParsedExpression> expression,
 	                            TableColumnType category);
+	DUCKDB_API ColumnDefinition(string name, LogicalType type, unique_ptr<ParsedExpression> expression,
```

Comment:
is `is_serial` necessary?
I imagine only the numeric types can be serial, no? Is there perhaps a way to add it to the LogicalType as ExtraTypeInfo, since it feels more part of the type of the column than the column itself.

Reply:
the existing PR for serial implementation did something similar if I recall correctly, and it just seemed like unnecessary - as I said, I followed the Postgres approach here, and we still need a way to be able to communicate to the whole tree this information without making the code a mess